src/partials/results.html

```diff
@@ -1,11 +1,15 @@
-<section class="lists">
+<section class="lists" ng-controller="searchResults()">
```

Comment:
Please don't try invoking your Controller... Probably won't work like you expect. Is this a different Controller from the one attached to the route?